src/main/java/org/broadinstitute/consent/http/db/mapper/DatasetMapper.java

```diff
@@ -26,6 +28,11 @@ public Dataset map(ResultSet r, StatementContext ctx) throws SQLException {
       if (hasColumn(r, "update_date")) {
           dataset.setUpdateDate(r.getTimestamp("update_date"));
       }
+      if (hasColumn(r, "dac_approval")) {
+        String boolString = r.getString("dac_approval");
+        Boolean value = Objects.isNull(boolString) ? null : Boolean.getBoolean(boolString);
```

Comment:
```suggestion
        Boolean value = Objects.isNull(boolString) ? null : r.getBoolean("dac_approval");
```
The current code is now handling null columns properly, but when `dac_approval` was true, `boolString` was `"t"`, so `Boolean.getBoolean("t")` was returning false